CRM-2322 - Revised Print Menu

lyricallytech opened the pull request:
Details for changes are within CRM-2322 ticket. This commit contains the updated file with the original "Print Menu" method deleted from the parent class and now includes the new "Print Menu".

lyricallytech committed:
CRM-2322 - Revised Print Menu

nflearga commented on Final_Project 2.py:
```diff
@@ -24,23 +24,19 @@ class Menu:
+        menu_item = ["Jerk Chicken Breast", "Rasta Pasta", "Oxtail", "Jerk Salmon", "Roti", "Rice & Peas", "Macaroni Pie", "Cabbage", "Mashed Potatoes", "Sweet Plantain", "Rum Punch", "Fruit Punch", "Peach Mango Juice", "Guava Pineapple", "Bottled Water"]
+        unit_price = [8.50, 12.00, 17.00, 13.00, 10.00, 6.00, 5.50, 3.50, 3.50, 3.50, 10.00, 5.50, 5.50, 5.50, 1.50]
+        menu_number = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]
+        menu_category = ["Main Dishes", "Side Dishes", "Beverages"]
```
This array should also have 15 elements, and the value is the Type for each of the products listed in the menu_item array.

lyricallytech committed:
CRM-2322 - Menu Categories Updated

lyricallytech committed:
Revised Version - Final Project

Revised version of final project that includes altered and condensed code for order intake and list printing functions.

lyricallytech committed:
Revision: Removal of IF/ELSE blocks

Revision of Final Project code that has removed the IF/ELSE blocks and thus making the program more dynamic rather than hard-coded.

lyricallytech committed:
Menu item entry not available.
